modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -172,8 +175,11 @@ The OTLP receiver ingests data using the OpenTelemetry protocol (OTLP).
           receivers: [otlp]
 ----
 <1> The OTLP gRPC endpoint. If omitted, the default `+0.0.0.0:4317+` is used.
-<2> The OTLP HTTP endpoint. If omitted, the default `+0.0.0.0:4318+` is used.
-<3> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<2> The server-side TLS configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<3> The path to the TLS certificate at which the server verifies a client certificate. This sets the `ClientCAs` and `ClientAuth` to `RequireAndVerifyClientCert` in the `TLSConfig`. For more information, see the link:https://godoc.org/crypto/tls#Config[documentation for the `Config` of the Golang TLS package].
+<4> Specifies the time interval at which the certificate is reloaded. If not set, the certificate is never reloaded. Accepts a string containing valid units of time such as `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```

Comment:
```suggestion
<4> Specifies the time interval at which the certificate is reloaded. If the value is not set, the certificate is never reloaded. `reload_interval` accepts a string containing valid units of time such as `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```

Reply:
Is this field mandatory? If yes, you can start the statement with `Required:`

Reply:
it's not mandatory